README.md

```diff
@@ -44,11 +44,24 @@ See the [Quickstart](https://salesforce.github.io/refocus/docs/01-quickstart.htm
 1. Create a new user record for yourself with your real email address, and set your profile to the `Admin` profile.
 1. If you want to restrict access to specific IP ranges, see [IP Restrictions](#ip-restrictions) below.
 1. If you want to use your own single sign-on (SSO) user authentication service, see [Authentication](#authentication) below.
-1. New users are assigned to the "RefocusUser" profile by default. This profile has read/write permission to all resource types. If you would like to restrict a class of users to have only read access to certain resource types, you can update the ____Access fields in the "RefocusUser" profile record.
-<!-- 1. Invite other users. Note: by default, only users with the `Admin` profile will be able to invite other users. If you want to let users register themselves as Refocus users, an Admin must set the config parameter `SELF_REGISTRATION_ENABLED` to `true`. -->
+1. New users are assigned to the "RefocusUser" profile by default. This profile has read/write permission to all resource types. If you would like to restrict a class of users to have only read access to certain resource types, you can update the ____Access fields in the "RefocusUser" profile record. <!-- 1. Invite other users. Note: by default, only users with the `Admin` profile will be able to invite other users. If you want to let users register themselves as Refocus users, an Admin must set the config parameter `SELF_REGISTRATION_ENABLED` to `true`. -->
 1. If you *only* want SSO users (no "Local Authentation" users), set environment variable `REJECT_LOCAL_USER_REGISTRATION` to `true`. This disables the `POST /v1/register` endpoint.
 1. Set environment variable `SESSION_SECRET` to sign the session ID cookie. (When deploying on Heroku, this will be generated automatically for you.)
 1. Set environment variable `SECRET_TOKEN` to create jwt tokens used for authentication. (When deploying on Heroku, this will be generated automatically for you.)
+1. You can automatically delete unused tokens based on a threshold you
+    define. Turn on this feature by defining two environment variables, 
+    one to set the schedule for the clock job and the other to set the
+    threshold for what to consider *unused*.
+    * Use environment variable name
+        `CLOCK_JOB_INTERVAL_deleteUnusedTokens` to set the frequency of
+        the clock job, i.e. how often should we check for unused tokens.
+        Set the value to a time offset like `12h` if you want to run the
+        clock job every twelve hours or `1d` if you want to run the job
+        once a day.
+    * Use environment variable name `DELETE_UNUSED_TOKENS_SINCE` to set
+        the threshold of what we should consider "unused". Set the value
+        to a negative time offset like `-30d` if you want to consider a token unused if it has
+        not been used in the last 30 days, or `-3m` for three months, etc.
```

Comment:
"3m" would be 3 minutes. ms doesn't have a month operator, but you can do "3w" or "1y".